### PR TITLE
Add contribution page

### PR DESCRIPTION
**Pull request by antho1404**
New `/community` page added that is synchronized with a database in firestore

### CODE REVIEW

**Comment by antho1404** [comment]
Deploy preview for *mesg* ready!

Built with commit 98e61bfcd0ae1b08e3278b14d5be351e1c4d3d4d

https://deploy-preview-179--mesg.netlify.com

**Comment by EmmanuelDrouin** [comment]
@antho1404 here is a list of things to do for the community page:

Community page
- [ ] Transform the community page into an index page (like the showcase)
- [ ] Link each contribution to the dedicated page
- [ ] Transform the embed-card section into a link to the dedicated contribution page

Contribution page
- [ ] Transform the contributor page into in _id page.
- [ ] Header - Replace the picture section by the Embed-card component (i would like to have a slot in the picture section of the header to do that. So in the future we can replace the picture by something else). The embed card section should be an external-link to the contribution.

Global to do list:

- [ ] Create Gitcoin account
- [ ] Add pagination component
- [ ] Integrate TypeForm
- [ ] Work on the content of the contribution page and how to improve it

**Comment by EmmanuelDrouin** [comment]
@antho1404 ready for review, everything is set

File: components/ContributionForm.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<template>
+  <Button
+    v-if="btnWhite"
```
Comment (antho1404) [inline components/ContributionForm.vue]: the `v-if` is not needed here you could simply use `v-bind="$attrs"` https://vuejs.org/v2/api/#vm-attrs
```html
<Button
    v-bind="$attrs"
    class="typeform-share button"
    href="https://mesgfoundation.typeform.com/to/bOJ0qz"
    data-mode="popup"
    data-submit-close-delay="0"
  >Add contribution</Button>
```

File: components/ContributionForm.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<template>
```
Comment (antho1404) [inline components/ContributionForm.vue]: I don't think this component is still used. If not, please remove.

Comment (EmmanuelDrouin) [reply components/ContributionForm.vue]: Yes right i will delete it

**Pull request merged**